modules/core/src/components/declarative-ui/xviz-metric.js

```diff
@@ -48,6 +49,7 @@ class XVIZMetricComponent extends PureComponent {
     streams: PropTypes.arrayOf(PropTypes.string).isRequired,
     title: PropTypes.string,
     description: PropTypes.string,
+    missingData: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Where is this used?

Reply:
Good catch.. no longer needed - Removing shortly